events/2025-05-12-galaxy-academy-2025.md

```diff
@@ -0,0 +1,257 @@
+---
+layout: event
+
+title: Galaxy Training Academy 2025
+description: |
+  The Galaxy Training Academy is a self-paced online training event for beginners and advanced learners who want to improve their Galaxy data analysis skills. 
+  Over the course of one week, we offer a diverse selection of learning track for you.
+
+# <button id="program-button" class="btn btn-info" onclick="$('#program-tab').tab('show');">Start the Course!</button>
+
+cover-image: events/images/galaxy-academy-logo.png
+cover-image-alt: logo for the Galaxy Academy event consisting of a laptop surrounded by illustrations of DNA molecules
+
+registration:
+  # link: 
+  deadline: 2025-05-08
+  open: false
+
+date_start: 2025-05-12
+date_end: 2025-05-16
+
+cost: free
+audience: Everyone who would like to get to know Galaxy and learn bioinformatics data analysis, whether you want to master a specific analysis or learn a new skill.
+contact_email: academy@galaxyproject.org
+
+async: true
+mode: online
+
+tags:
+- microbiome
+- single-cell
+- proteomics
+- introduction
+- galaxy-interface
+- assembly
+- statistics
+
+contributions:
+    organisers:
+        - delphine-l
+        - teresa-m
+        - scottcain
+        - natalie-wa
+        - annasyme
+        - shiltemann
+        - dianichj
+        - dadrasarmin
+    instructors:
+        - ahmedhamidawan
+        - erxleben
+        - annasyme
+        - anuprulez
+        - abretaud
+        - bebatut
+        - bgruening
+        - clsiguret
+        - dannon
+        - dianichj
+        - deeptivarshney
+        - delphine-l
+        - elichad
+        - EngyNasr
+        - emmaustin20
+        - GarethPrice-Aus
+        - igormakunin
+        - jdavcs
+        - lldelisle
+        - bernt-matthias
+        - foellmelanie
+        - meltemktn
+        - mschatz
+        - natalie-wa
+        - natefoo
+        - paulzierep
+        - pavanvidem
+        - plushz
+        - pratikdjagtap
+        - RZ9082
+        - rlibouba
+        - SaimMomin12
+        - stephanierobin
+        - subinamehta
+        - teresa-m
+        - timothygriffin
+        - tcollins2011
+        - nomadscientist
+        - wm75
+    funding:
+        - eurosciencegateway
+        - biont
+        - nfdi4plants
+        - deNBI
+        - elixir-europe
+        - mwk
+        - abromics
+        - ifb
+
+location:
+  name: online
+
+infrastructure:
+  tiaas: true
```

Comment:
are any of the servers using TIaaS? if so, give a link below, if not, just remove this or set to false to hide the TIaaS section from the setup instructions